code/script.js

```diff
@@ -1,51 +1,113 @@
 // Variables that point to selected DOM elements
-const chat = document.getElementById('chat');
-
-// If you need any global variables that you can use across different functions, declare them here:
-
-
-// Declare your functions after this comment
-
-// This function will add a chat bubble in the correct place based on who the sender is
-const showMessage = (message, sender) => {
-  // the if statement checks if the sender is 'user' and if that's the case it inserts an html senction inside the chat with the posted message
-  if (sender === 'user') {
-    chat.innerHTML += `
-      <section class="user-msg">
-        <div class="bubble user-bubble">
-          <p>${message}</p>
-        </div>
-        <img src="assets/user.png" alt="User" />  
-      </section>
-    `
-    // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
-  } else if (sender === 'bot') {
-    chat.innerHTML += `
-      <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
-        <div class="bubble bot-bubble">
-          <p>${message}</p>
-        </div>
-      </section>
-    `
+document.addEventListener('DOMContentLoaded', ()=> {
+  //const chat = document.getElementById('chat');
+  const nameForm = document.getElementById('name-form');
+  let userInput = '';
```

Comment:
We see that you med userInput global (learned the tearm today) so that i can picked up in all the functions. Great!